TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -485,23 +471,6 @@ TYPO3:
     view:
       title: 'Neos Setup'
 
-  Media:
-    thumbnailPresets:
-      'TYPO3.Neos:Thumbnail':
-        maximumWidth: 250
-        maximumHeight: 250
-      'TYPO3.Neos:Preview':
-        maximumWidth: 1000
-        maximumHeight: 1000
```

Comment:
Would love a discussion about this, because it's currently confusing how and where this is used.